modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/service/test/JournalFolderLocalServiceTest.java

```diff
@@ -51,25 +54,38 @@ public class JournalFolderLocalServiceTest {
 	@Before
 	public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
+
+		_serviceContext = ServiceContextTestUtil.getServiceContext(
+			_group.getGroupId());
 	}
 
 	@Test
-	public void testGetFoldersAndArticlesCount() throws Exception {
-		JournalFolderFixture journalFolderFixture = new JournalFolderFixture(
-			_journalFolderLocalService);
+	public void testFolderStatusAfterAddAndUpdate() throws Exception {
```

Comment:
@ak-ragnor , it would be better to have two tests for this, although one would seem enough. Don't use the TestPropsValues.getUser for add user, better create a new user just like you did with update